src/test/java/teammates/test/cases/ui/browsertests/StudentCourseJoinConfirmationPageUiTest.java

```diff
@@ -70,12 +71,17 @@ private void testJoinNewConfirmation() {
         studentHomePage = createCorrectLoginPageType(browser.driver.getPageSource())
                            .loginAsStudent(TestProperties.inst().TEST_STUDENT1_ACCOUNT,
                                                   TestProperties.inst().TEST_STUDENT1_PASSWORD);
-        studentHomePage.verifyStatus(
-                String.format(Const.StatusMessages.STUDENT_COURSE_JOIN_SUCCESSFUL, courseId));
+        
+        String expectedStatus = String.format(Const.StatusMessages.STUDENT_COURSE_JOIN_SUCCESSFUL, "[" + courseId + "] " + courseName) + '\n' 
+                                + String.format(Const.StatusMessages.HINT_FOR_NO_SESSIONS_STUDENT, "[" + courseId + "] " + courseName) + '\n'  
+                                + "Meanwhile, you can update your profile here.";
```

Comment:
Yes the test passes. The value is different as I did not include the <a></a> tags in the expected status message. AppPage.verifyStatus calls getText() on the statusMessage, which returns the innerText of the element. The html tags (<a href ...>) in the status message are not included in the innerText. 

The status message is tested again in StudentCourseJoinAuthenticatedActionTest, this time with the <a></a> tag included.